rpi3.mk

```diff
@@ -31,13 +32,21 @@ RPI3_BOOT_CONFIG	?= $(RPI3_FIRMWARE_PATH)/config.txt
 RPI3_UBOOT_ENV		?= $(ROOT)/out/uboot.env
 RPI3_UBOOT_ENV_TXT	?= $(RPI3_FIRMWARE_PATH)/uboot.env.txt
 RPI3_STOCK_FW_PATH	?= $(ROOT)/firmware
-RPI3_STOCK_FW_PATH_BOOT	?= $(RPI3_STOCK_FW_PATH)/boot
+ifeq ($(PLATFORM),rpi3-b)
+RPI3_STOCK_FW_PATH_BOOT ?= $(RPI3_STOCK_FW_PATH)/boot
+else
+RPI3_STOCK_FW_PATH_BOOT ?= $(RPI3_FIRMWARE_PATH)/$(PLATFORM)
+endif
 OPTEE_BIN		?= $(OPTEE_PATH)/out/arm/core/tee-header_v2.bin
 OPTEE_BIN_EXTRA1	?= $(OPTEE_PATH)/out/arm/core/tee-pager_v2.bin
 OPTEE_BIN_EXTRA2	?= $(OPTEE_PATH)/out/arm/core/tee-pageable_v2.bin
 
 LINUX_IMAGE		?= $(LINUX_PATH)/arch/arm64/boot/Image
-LINUX_DTB		?= $(LINUX_PATH)/arch/arm64/boot/dts/broadcom/bcm2710-rpi-3-b.dtb
+ifeq ($(PLATFORM),rpi3-b)
+LINUX_DTB               ?= $(LINUX_PATH)/arch/arm64/boot/dts/broadcom/bcm2710-rpi-3-b.dtb
+else
+LINUX_DTB               ?= $(LINUX_PATH)/arch/arm64/boot/dts/broadcom/bcm2710-rpi-3-b-plus.dtb
```

Comment:
This is good, but I noticed that the same dtb is showing up on several places, which might be a bit confusing. We should try clean that up at some point in the future.
```bash
$ find . -name "bcm2710-rpi-3-b.dtb"
./build/rpi3/firmware/rpi3-bplus/bcm2710-rpi-3-b.dtb
./linux/arch/arm64/boot/dts/broadcom/bcm2710-rpi-3-b.dtb
./firmware/boot/bcm2710-rpi-3-b.dtb
./out-br/target/boot/bcm2710-rpi-3-b.dtb

$ find . -name "bcm2710-rpi-3-b-plus.dtb"
./build/rpi3/firmware/rpi3-bplus/bcm2710-rpi-3-b-plus.dtb
./linux/arch/arm64/boot/dts/broadcom/bcm2710-rpi-3-b-plus.dtb
```